draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -265,6 +268,23 @@ Throughout the operations of this signature scheme, each message that is signed
 
 Aside from the message generators, the scheme uses two additional generators: `Q_1` and `Q_2`. The first (`Q_1`), is used for the blinding value (`s`) of the signature. The second generator (`Q_2`), is used to sign the signature's domain, which binds both the signature and generated proofs to a specific context and cryptographically protects any potential application-specific information (for example, messages that must always be disclosed etc.).
 
+### Serializing to octet strings
+
+When serializing different types of elements (i.e., Points, Scalars etc.) to get their octet representation, each element will be serialized with a specific operation, depending on its type. More concretely,
+
+- Points in `G*` will be serialized using the `point_to_octets_g*` implementation for a particular ciphersuite.
+- Non-negative integers will be serialized using `I2OSP` with an output length of 8 bytes.
+- Scalars will be serialized using `I2OSP` with a constant output length defined by a particular ciphersuite.
+- ASCII strings will be transformed into octet strings using UTF-8 encoding.
```

Comment:
```
- ASCII strings will be transformed into octet strings using UTF-8 encoding.
```

This reads a little strangely as there is no transformation involved, ASCII strings are UTF-8 strings.

Reply:
I do agree with this. The current rule followed in the doc, is that ASCII string refers to text and ASCII code refers to bytes (note the ciphersuite id and DSTs encodings).

We could make this clearer by updating ASCII string -> Text or ASCII text or String??

Since this affects the whole doc, IMO its better to open a issue for tracking and address it in a separate PR?

Reply:
Opened #234 to track this issue!